demo/index.html

```diff
@@ -89,6 +89,42 @@
     </div>
 
     <div style="max-width: 800px; margin: auto;" class="vertical-section-container centered">
+        <h3>kwc-challenge-set disabled</h3>
+        <demo-snippet>
+            <template id="four" is="dom-bind">
+                <kwc-challenge-set challenge-set="[[challengeSet]]" disabled$="[[disabled]]"></kwc-challenge-set>
+
+                <!--
```

Comment:
Is this comment just to demonstrate the shape of the data in the demo snippet?

Reply:
I don't know, I just repeated the pattern I saw on the other demo snippets.